lib/bakeAmbientOcclusion.js

```diff
@@ -39,6 +39,34 @@ var cartesian2Scratch = new Cartesian2();
 var quaternionScratch = new Quaternion();
 var matrix3Scratch = new Matrix3();
 var matrix4Scratch = new Matrix4();
+var trianglePixelMarchScratch = {
+    uv0 : cartesian2Scratch,
```

Comment:
No need to include the `Scratch` prefix here since it is already in `trianglePixelMarchScratch`; however, I don't know that I would use an object literal here since property access is likely to be slow.  See https://www.youtube.com/watch?v=XAqIpGU8ZZk